app/models/state_file_md_intake.rb

```diff
@@ -248,6 +248,26 @@ def qualifies_for_pension_exclusion?(filer)
     send("#{filer}_senior?".to_sym) || at_least_one_disabled_filer_with_proof?
   end
 
+  def has_filer_under_65?
+    if filing_status_mfj?
+      !primary_senior? || !spouse_senior?
+    else
+      !primary_senior?
+    end
+  end
+
+  def no_proof_of_disability_submitted?
+    if filing_status_mfj?
+      if primary_proof_of_disability_submitted_yes? || spouse_proof_of_disability_submitted_yes?
```

Comment:
i *believe* you don’t need the extra case for mfj since this is an or statement.

Reply:
& why do you use yes here instead of no?